locales-pending/settings-premium.ftl

```diff
@@ -63,3 +63,97 @@ settings-delete-monitor-plus-account-dialog-cancel-button-label = Never mind, ta
 ## Monthly Monitor Report
 
 settings-alert-preferences-allow-monthly-monitor-plus-report-title = Monthly { -brand-monitor-plus } report
+
+## Update scan info
+
+settings-update-scan-info-description = { -brand-monitor } is most effective at protecting your info when you add specific details. Add any of your name variations, emails, or locations. <a>Why should I add details for my scan?</a>
+settings-update-scan-info-upsell-cta-label = Upgrade to { -brand-monitor-plus } to protect your personal info
+settings-details-saved-notification-title = Details saved
+settings-details-saved-notification-message = Thanks for helping { -brand-monitor } protect you. We’ll use this info in future scans.
+
+## Details about you
+
+settings-details-about-you-header = Details about you
+settings-details-about-you-description = { -brand-monitor } will use this info for scans and data removal requests.
+settings-details-about-you-name-label = Name
+settings-details-about-you-date-of-birth-label = Date of birth
+settings-details-about-you-phone-label = Phone number
+settings-details-about-you-location-label = Location
+settings-details-about-you-cta-label = Add details
```

Comment:
Can we use `button` instead of `cta` in these IDs?

Reply:
Yes. Updated the IDs as part of https://github.com/mozilla/blurts-server/pull/5793/commits/338a0fd0bc12d6d437bc410e39e70b98103e7b41.